astropy/nddata/nddata.py

```diff
@@ -140,32 +139,32 @@ def __init__(self, data, uncertainty=None, mask=None, wcs=None,
             # other parameters.
             if (unit is not None and data.unit is not None and
                     unit != data.unit):
-                log.info("Overwriting NDData's current "
-                         "unit with specified unit")
+                log.info("overwriting NDData's current "
```

Comment:
Huh... never noticed that before. Thanks for clarifying!
```python
>>> float('dsadas')
ValueError: could not convert string to float: dsadas
```

*However*, it seems to go against [English grammar](https://www.grammarly.com/handbook/mechanics/capitalization/16/capitalization-first-word-after-a-colon-if-it-begi/)?